homeassistant/components/lutron_caseta/strings.json

```diff
@@ -1,5 +1,21 @@
 {
   "config": {
-    "title": "Lutron Caséta"
+    "title": "Lutron Caséta",
+    "step": {
+      "user": {
```

Comment:
I think we should remove the user step strings since we don't implement that yet.